src/concurrently.js

```diff
@@ -79,6 +86,7 @@ function mapToCommandInfo(command) {
         command: command.command || command,
         name: command.name || '',
         prefixColor: command.prefixColor || '',
```

Comment:
we can get rid of this line now (and all its wiring)

Reply:
I did that. Thanks!